src/cudamatrix/cu-kernels.cu

```diff
@@ -3666,11 +3666,11 @@ void _cuda_mat_copy_range_clamped(
       // compute offset row
       int32_t r_in = r + row_start;
       // clamp if necessary
-      if (r_in < clamp_low) r_in = clamp_low;
-      if (r_in > clamp_high) r_in = clamp_high;
+      if (r_in < clamp_low) r_in=clamp_low;
```

Comment:
changes around these lines seem to only make it non-google-style-guide-compliant.

Reply:
sorry looks like a case of a merge error on my side. i'll fix and push.